### PR TITLE
LiteralColorExpression tests

### PR DESCRIPTION
**Pull request by ggetz**
* Renamed `ConstantColorExpression` to `LiteralColorExpression`
* Wrote Spec for `LiteralColorExpression`

### CODE REVIEW

File: Source/Scene/Cesium3DTileStyle.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ define([
        './BooleanRegularExpression',
        './ColorRampExpression',
        './ColorMapExpression',
-       './ConstantColorExpression',
+       './LiteralColorExpression',
```
Comment (pjcozzi) [inline Source/Scene/Cesium3DTileStyle.js]: Search and replace throughout this file (and then test in Sandcastle to double check).

File: Specs/Scene/LiteralColorExpressionSpec.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    'use strict';
+
+    function MockStyleEngine() {
+    }
+
+    MockStyleEngine.prototype.makeDirty = function() {
+    };
+
+    it('constructs with hex', function() {
+        var expression = new LiteralColorExpression(new MockStyleEngine(), "#ffffff");
```
Comment (pjcozzi) [inline Specs/Scene/LiteralColorExpressionSpec.js]: Use single quotes for all the strings in this file.  We usually always use single except for JSON examples, e.g., the CZML examples in Sandcastle, and the style examples we have been making in Sandcastle.

**Comment by pjcozzi** [comment]
Check for the `DeveloperError`s.  For example, https://github.com/ggetz/cesium/blob/literal-color-exp-tests/Specs/Core/Cartesian2Spec.js#L42-L46

**Comment by pjcozzi** [comment]
>          if (color === undefined) {
             throw new DeveloperError('color must be defined');
         }

Use `!defined` (same comment throughout).

Also, looking more carefully at this, the string should be "color must be a valid CSS string" despite me saying something else previously.  Sorry.

**Comment by pjcozzi** [comment]
Nice work!  Just those comments.

**Comment by ggetz** [comment]
I made the updates and added the test to check for the `DeveloperError`. Also I removed the DeveloperError in the setter for `color` because it sets the color directly, not by constructing it with a css string.

**Comment by pjcozzi** [comment]
Thanks!

**Pull request merged**